src/constructs/core/identity.ts

```diff
@@ -13,7 +13,7 @@ export interface AppIdentity {
 export interface Identity extends StackStageIdentity, AppIdentity {}
 
 export const AppIdentity = {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types -- user defined type guard
```

Comment:
eslint was reporting `@typescript-eslint/explicit-module-boundary-types` as "rule is disabled but never reported". I can't quite understand what that means, but removing the exception solved the issue 🤷🏽‍♂️ .